### ISSUE TITLE
Versioning

### ISSUE THREAD

**Issue by Barina** [Barina/Dark-Telegram#11]
I feel like the versioning of the theme is too much in the grey area.
And I want to keep it simple so from now on we'll use the simple general formula like this
[major].[minor].[patch]
Where major is for big changes including rewriting the theme from scratch.
The minor reserved for small to medium changes like new features.
And the patch if for fixing issues only.
Every part will be incremental and won't resets to 0 or go lower than what its already is.
If a release includes both a new feature and a fix, the minor will increment by 1 as well as the patch.

I think this is pretty easy to follow and enough to make everybody understand what the release contains without diving to the changelog. :)

This is an open conversation feel free to suggest better alternatives :D

**Comment by prakharb5** [comment]
Whenever there is a new major release, the minor and patch versions should become 0, not stay as it is. Otherwise, it can be confusing.

**Comment by Barina** [comment]
I agree the major should reset the minor & patch, we didn't had a major release yet but I know one is coming (they working on a new website -_-)
We also can decide to reset the patch when there's a minor release to avoid big numbers in the future...
